### ISSUE TITLE
zombie process is increased

### ISSUE THREAD

**Issue by zoneofjin** [fluent/fluentd-ui#165]
I use fluend-ui

Started with the following command:

[root@localhost ~]# fluentd-ui start

When the fluentd start in Dashbard web,
zombie process is increased.

[root@localhost ~]# ps aux | grep ruby

root      5541  8.2  0.0      0     0 pts/1    Z+   14:22   0:00 [ruby] <defunct>
root      5541  1.0  0.0      0     0 pts/1    Z+   14:22   0:00 [ruby] <defunct>
root      5593  2.7  0.0      0     0 pts/1    Z+   14:23   0:00 [ruby] <defunct>

Please let me know the reason why it occurs.

**Comment by zoneofjin** [comment]
- CentOS 6.5
- ruby 2.2.0p0 (2014-12-25 revision 49005) [x86_64-linux]

**Comment by uu59** [comment]
I've confirmed that.

I've run `fluentd-ui start` with ruby 2.2.0p0/CentOS 6.5 on [this AMI](https://aws.amazon.com/marketplace/pp?sku=eggbgx9svw4xhzs1omttdv29q), then "start" and "stop" some times from Dashboard. Then processes are as below:

```console
[root@ip-10-120-88-237 ~]# ps afux|grep ruby
root      1514  1.2  3.1 204968 18896 pts/0    Sl+  08:11   0:00  |       \_ /root/.rbenv/versions/2.2.0/bin/ruby /root/.rbenv/versions/2.2.0/bin/fluentd-ui start
root      1543 30.4 21.9 847720 132848 pts/0   Sl+  08:11   0:12  |           \_ /root/.rbenv/versions/2.2.0/bin/ruby /root/.rbenv/versions/2.2.0/bin/rackup  --pid /root/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/fluentd-ui-0.3.16/tmp/fluentd-ui.pid -p 9292 -E production /root/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/fluentd-ui-0.3.16/config.ru
root      1644  0.0  0.1 103240   868 pts/1    S+   08:12   0:00          \_ grep ruby
root      1628  0.0  2.6 203436 15796 ?        Sl   08:12   0:00 /root/.rbenv/versions/2.2.0/bin/ruby /root/.rbenv/versions/2.2.0/bin/fluentd --use-v1-config -c /root/.fluentd-ui/fluent.conf -d /root/.fluentd-ui/fluent.pid -o /root/.fluentd-ui/fluent.log
root      1631  3.7  3.9 490216 23808 ?        Sl   08:12   0:00  \_ /root/.rbenv/versions/2.2.0/bin/ruby /root/.rbenv/versions/2.2.0/bin/fluentd --use-v1-config -c /root/.fluentd-ui/fluent.conf -d /root/.fluentd-ui/fluent.pid -o /root/.fluentd-ui/fluent.log
[root@ip-10-120-88-237 ~]# ps afux|grep Z
USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root      1587  1.5  0.0      0     0 pts/0    Z+   08:12   0:00  |               \_ [fluentd] <defunct>
root      1622  2.2  0.0      0     0 pts/0    Z+   08:12   0:00  |               \_ [fluentd] <defunct>
root      1658  0.0  0.1 103240   868 pts/1    S+   08:12   0:00          \_ grep Z
```

these zombies are flushed after fluentd-ui stopped.

I'll investigate it deeply later, quickly thinking is that launching fluentd process problem.

**Comment by zoneofjin** [comment]
Thank you for your help. I just wonder when it can be solved?

**Comment by uu59** [comment]
I hope as soon as possible :)
Sorry for inconvenience

**Comment by yoshihara** [comment]
I fixed it by #166 and released [v0.3.17](https://github.com/fluent/fluentd-ui/commit/3d8e1cb5c6252a4c4e26fcbb444554cc90c7b8b4 ). please try it :)
